chatsky/stats/instrumentor.py

```diff
@@ -160,7 +160,7 @@ async def __call__(self, wrapped, _, args, kwargs):
         pipeline_component = get_extra_handler_name(info)
         attributes = {
             "context_id": str(ctx.id),
-            "current_turn_id": ctx.current_turn_id,
+            "turn_id": ctx.current_turn_id,
```

Comment:
This needs to be renamed in all stats-related files.
The naming should be the following:

1. `current_turn_id` inside `Context` (it always returns the current turn id, is incremented every turn);
2. `turn_id` everywhere else (we save `current_turn_id` as `turn_id` every time we extract something).

Essentially, this task is "rename request_id to turn_id" instead of "rename request_id to current_turn_id".